<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mallet_GUIClass</class>
 <widget class="QWidget" name="Mallet_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mallet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mallet_icon.png</normaloff>mallet_icon.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="templates">
    <attribute name="title">
     <string>Templates</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="ipLine">
       <property name="placeholderText">
        <string>Listener IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLine">
       <property name="placeholderText">
        <string>Listener Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="shellRadio">
       <property name="text">
        <string>Shell</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="meterpreterRadio">
       <property name="text">
        <string>Meterpreter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="CustomShellcode">
    <attribute name="title">
     <string>Custom Shellcode</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="fileButton">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="fileLine"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Dropper</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="optionsBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Payload Options</string>
   </property>
   <widget class="QLabel" name="placementLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Placement</string>
    </property>
   </widget>
   <widget class="QComboBox" name="placementCombo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.data</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.rsrc</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="encLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption Chain</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="encLine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>179</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>123123123...</string>
    </property>
   </widget>
   <widget class="QLabel" name="encLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1. AES-256</string>
    </property>
   </widget>
   <widget class="QLabel" name="encLabel_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2. RC4</string>
    </property>
   </widget>
   <widget class="QLabel" name="encLabel_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3. XOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="formatLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="executeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Execution Method</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatCombo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.exe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.dll</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="executeCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Local Thread</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Remote Process Injection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Remote DLL Injection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Remote Thread Hijack</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="exportLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Export name for .dll</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exportLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>EvilFunction</string>
    </property>
   </widget>
   <widget class="QLabel" name="processLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="processLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>notepad.exe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quickModeButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>mallet_splash.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mallet_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
